common/menu.c

```diff
@@ -406,6 +440,33 @@ void drawTime() {
 
 }
 
+void drawCharge() {
+
+    char chargeString[5];
+
+    uint32_t batteryCharge;
+    ChargerType chargeType = ChargerType_None;
```

Comment:
Breaks pc-build.

Reply:
How does changing this to the following looks?
```
    #ifdef __SWITCH__
    ChargerType chargeType = ChargerType_None;
    if (psmInitialized)
    {
        psmGetBatteryChargePercentage(&batteryCharge);
        psmGetChargerType(&chargeType);
    }
    #else
    batteryCharge = 100;
    #endif

    ...

    #ifdef __SWITCH__
    if (chargeType > (ChargerType)ChargerType_None)
    #else
    if (false) // No action for non-switch devices
    #endif
        drawImage(tmpX - 20, 0 + 47 + 10 + 5, 9, 15, charging_icon_small, IMAGE_MODE_RGBA32);
```